lib/signs/bus.ex

```diff
@@ -612,6 +670,36 @@ defmodule Signs.Bus do
     end
   end
 
+  defp bridge_tts_audio(bridge_status, bridge_enabled?, current_time, state) do
+    %{chelsea_bridge: chelsea_bridge} = state
+
+    if bridge_enabled? && chelsea_bridge &&
+         bridge_status_raised?(bridge_status, current_time) do
+      {duration, duration_spanish} =
+        case bridge_status_minutes(bridge_status, current_time) do
+          minutes when minutes < 2 ->
+            {"We expect it to be lowered soon.", "Esperamos que baje pronto."}
+
+          minutes ->
+            {"We expect this to last for at least #{minutes} more minutes.",
+             "Esperamos que esto dure al menos #{minutes} minutos más."}
+        end
+
+      english_text =
+        "The Chelsea Street bridge is raised. #{duration} SL3 buses may be delayed, detoured, or turned back."
+
+      spanish_text =
+        "Se levanta el puente de Chelsea Street. #{duration_spanish} Los autobuses SL3 pueden sufrir retrasos, desvíos o dar marcha atrás."
```

Comment:
On the topic of conjugations, this should probably be `se levante` — I think "se levant**a** el puente" means something like "the bridge raises" (is currently going up).

I plugged this into DeepL as well and it suggested _"El puente de Chelsea Street está levantado"_, which might be a more natural construction. `levantado` is the past participle (adjective) form of the verb, making this a fairly direct translation: <ruby>The bridge<rt>El puente</rt></ruby> / <ruby>of<rt>de</rt></ruby> / Chelsea Street / <ruby>is<rt>está</rt></ruby> / <ruby>raised<rt>levantado</rt></ruby>.